modules/nw-network-config.adoc

```diff
@@ -25,6 +25,11 @@ Phase 1:: After entering the `openshift-install create install-config` command.
 +
 For more information on these fields, refer to "Installation configuration parameters".
 
+[NOTE]
+====
+Set `networking.machineNetwork` to match the CIDR which the preferred NIC resides.
```

Comment:
The indenting for "Phase 1" was already a little off before this PR (because of the unordered list in the middle of it), but the new `[NOTE]` kinda highlights it a bit more. Suggest the following use of an [open block](https://docs.asciidoctor.org/asciidoc/latest/blocks/open-blocks/) wrapped around the list (with an additional `+` before the `[NOTE]`) to get things lined up in what seems like the original intention:

```
Phase 1:: After entering the `openshift-install create install-config` command. In the `install-config.yaml` file, you can customize the following network-related fields:
+
--
* `networking.networkType`
* `networking.clusterNetwork`
* `networking.serviceNetwork`
* `networking.machineNetwork`
--
+
For more information on these fields, refer to "Installation configuration parameters".
+
[NOTE]
====
Set the `networking.machineNetwork` field to match the CIDR which the preferred NIC resides.
====
```

Result:

![image](https://user-images.githubusercontent.com/3442316/116121837-42b23f00-a67e-11eb-8459-7d32cf17dbc3.png)